eos/b-decays/b-to-3l-nu_TEST.cc

```diff
@@ -40,18 +40,17 @@ class BToThreeLeptonsNeutrinoTest :
 
             {
 
-                BToThreeLeptonsNeutrino three_lnu(p, Options{ });
+                BToThreeLeptonsNeutrino three_lnu(p, Options{ { "l", "mu" }, { "lprime", "e" } });
 
                 // Tested against an IPython/Jupyter notebook implementation of Stephan Kürten
-                //TODO: do these tests for the branching ratio instead of decay width since I dont export decay widths anymore
-                TEST_CHECK_NEARLY_EQUAL(4.4169722387470174e-11, three_lnu.double_differential_branching_ratio(1.2, 0.6), 1e-15);
-                TEST_CHECK_NEARLY_EQUAL(4.4899433542534617e-11, three_lnu.double_differential_branching_ratio(1.2, 0.9), 1e-15);
-                TEST_CHECK_NEARLY_EQUAL(1.9615874219144387e-11, three_lnu.double_differential_branching_ratio(1.5, 0.6), 1e-15);
-                TEST_CHECK_NEARLY_EQUAL(1.991304494545849e-11 , three_lnu.double_differential_branching_ratio(1.5, 0.9), 1e-15);
-                TEST_CHECK_NEARLY_EQUAL(7.978355662846529e-09 , three_lnu.double_differential_branching_ratio(0.6, 1.2), 1e-13);
-                TEST_CHECK_NEARLY_EQUAL(1.6747424093160892e-10, three_lnu.double_differential_branching_ratio(0.9, 1.2), 1e-14);
-                TEST_CHECK_NEARLY_EQUAL(8.170238677828805e-09 , three_lnu.double_differential_branching_ratio(0.6, 1.5), 1e-13);
-                TEST_CHECK_NEARLY_EQUAL(1.7053307009828605e-10, three_lnu.double_differential_branching_ratio(0.9, 1.5), 1e-14);
+                TEST_CHECK_NEARLY_EQUAL(4.39594668126115e-11  , three_lnu.double_differential_branching_ratio(1.2, 0.6), 1e-24);
```

Comment:
In a separate commit, please invert the order of the first and second argument of ``TEST_CHECK_NEARLY_EQUAL`` in this file.